client/analytics/filters/index.js

```diff
@@ -0,0 +1,84 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { Fragment } from '@wordpress/element';
+import { noop } from 'lodash';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import AdvancedFilters from 'components/advanced-filters';
+import Card from 'components/card';
+import DatePicker from 'components/date-picker';
+import FilterPicker from 'components/filter-picker';
+import { H, Section } from 'layout/section';
+import './style.scss';
+
+const ReportFilters = ( { advancedConfig, filters, filterPaths, query, path } ) => {
+	let advancedCard = false;
+	switch ( query.filter ) {
```

Comment:
We have `FILTER_PARAM` defined in `filter-picker`, but it should always be `filter` - I think it would be more clear to use `query.filter` directly. 

If we _do_ need to use constants for query params, maybe we should collect them somewhere (in the nav-utils library?) to make sure we're not overlapping anywhere?